content/en/docs/tasks/run-application/run-single-instance-stateful-application.md

```diff
@@ -49,14 +49,19 @@ Note: The password is defined in the config yaml, and this is insecure. See
 for a secure solution.
 
 {{< code file="mysql-deployment.yaml" >}}
+{{< code file="mysql-pv.yaml" >}}
+
+1. Deploy the pv and pvc of the YAML file:
```

Comment:
nit: we avoid use 'pv' or 'pvc' in documentation text, pls consider use 'PV' and 'PVC' instead, or better yet, PersistentVolume and PersistentVolumeClaim respectively.